superset-frontend/src/common/components/common.stories.tsx

```diff
@@ -0,0 +1,53 @@
+/**
+ * Licensed to the Apache Software Foundation (ASF) under one
+ * or more contributor license agreements.  See the NOTICE file
+ * distributed with this work for additional information
+ * regarding copyright ownership.  The ASF licenses this file
+ * to you under the Apache License, Version 2.0 (the
+ * "License"); you may not use this file except in compliance
+ * with the License.  You may obtain a copy of the License at
+ *
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing,
+ * software distributed under the License is distributed on an
+ * "AS IS" BASIS, WITHOUT WARRANTIES OR CONDITIONS OF ANY
+ * KIND, either express or implied.  See the License for the
+ * specific language governing permissions and limitations
+ * under the License.
+ */
+import React from 'react';
+import { action } from '@storybook/addon-actions';
+// import { withKnobs, select, boolean, text } from '@storybook/addon-knobs';
+import Modal from './Modal';
+import Tabs from './Tabs';
+
+export default {
+  title: 'Common Components',
+  // decorators: [withKnobs],
```

Comment:
It might be worth adding "knobs" to these stories, so I left these comments here, but the stories are super simple for the moment.

Reply:
nit: commented out code bothers me. It's in a storybook, though, so I won't strongly object.

Reply:
I was figuring @riahk or I would probably add some knobs to these stories, so I left the scaffolding there. I'll throw one or two in for good measure.